Fix MC-117075: TE Unload Lag Spike

zachbr opened the pull request:
PaperMC port of https://github.com/MinecraftForge/MinecraftForge/pull/4281

Commit:
Fix MC-117075: TE Unload Lag Spike

PaperMC port of https://github.com/MinecraftForge/MinecraftForge/pull/4281

kashike commented on Spigot-Server-Patches/0232-Fix-MC-117075-TE-Unload-Lag-Spike.patch:
```diff
@@ -0,0 +1,101 @@
+         }
+         // CraftBukkit end */
+ 
++        this.removeTileEntitiesForRemovedChunks(); // Paper - Fix MC-117075: TE Unload Lag Spike
```
Should go above `this.O = false;`

Commit:
Start using Travis-CI again

The only way to get Paper to build, specifically past the jar remap
step, is to use a macOS host. Don't ask me why, I'm just the messenger.
